recipes/send2trash/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "Send2Trash" %}
+{% set version = "1.4.2" %}
+{% set sha256 = "725fbce571dffe0b640e2f1788d52c3c544b510f9d8f69b2597c8c2555bc8441" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - python
+  imports:
+    - send2trash
+  commands:
+    - echo test > test.txt
+    - python -c "from send2trash import *; send2trash('test.txt')"
+
+about:
+  home: https://github.com/hsoft/send2trash
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ RECIPE_DIR }}/LICENSE'
```

Comment:
I can't tell from the logs whether the `license_file` is being picked up. This line differs from what is described in the docs:

```
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Reply:
Updated!